.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Describe the Problem and Solution
+<!-- Write 2-3 sentences summarising the reasons behind this change (this might be the problem you're solving, or the context around the request) and the solution you take. !-->  
+
+## What makes this change safe?
+<!-- DELETE WHEN COMPLETED
+A good answer to this question helps the reviewers understand where they should focus their attention, consider these questions:
+- Is the change risky or not? Why? 
+- What tests are you adding? Why?
+- What existing tests are you relying on? 
+- What are you worried about that you'd like the reviewer to focus on?
+References:
+- [Risk level guide](https://github.com/wiredtiger/wiredtiger/blob/develop/.github/risk_level_guide.md)
+- [Testing frameworks](https://github.com/wiredtiger/wiredtiger/blob/develop/.github/testing_frameworks.md)
+!-->
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
+- [ ] I have made corresponding changes to the documentation (if applicable).
+- [ ] I have added/updated tests that demostrate my fix is effective or that my feature works correctly.
```

Comment:
There's a typo in `demostrate`.

Reply:
Good catch! Fixed in [cde76c0](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/cde76c03a3da2877ed66beb78c12876859a10939)